lib/runner/TestRunner.js

```diff
@@ -160,6 +168,42 @@ module.exports = class TestRunner {
         return testResults;
     }
 
+    async saveTestResults(testResults, testSuite, startTimeTestSuite) {
+        const urlString = process.env.SOURCE_API_BASE_URL ? process.env.SOURCE_API_BASE_URL : "https://source-api.bespoken.tools";
+        const urlParts = url.parse(urlString);
+        const token =  process.env.SOURCE_API_ACCESS_TOKEN || "";
+
+        const options = {
+            headers: {
+                "Content-Type": "application/json",
+                "x-access-token": token,
+            },
+            host: urlParts.hostname,
+            method: "POST",
+            path: "/v2/testResultsHistory",
+            port: urlParts.port,
+            protocol: urlParts.protocol,
+        };
+
+        const { projectId, fileName, virtualDeviceToken } = testSuite;
+        const runId = uniqid();
```

Comment:
Test runner runs for each file, this means we would be getting different id's for the different records belonging to the same run. The runId should most likely be set up as part as the Configuration object that is common to all runs, and then can be retrieved here in order for each file to have the same id as the others